analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -283,6 +283,11 @@ def main(args):
         LOG.error(fnerr)
         sys.exit(1)
 
+    for input_path in args.input:
+        if not os.path.exists(input_path):
+            LOG.error(f"Input path {input_path} does not exist!")
+            sys.exit(1)
```

Comment:
You could consider emitting a warning for all non-existent paths before exiting.